conf/server/server.conf

```diff
@@ -19,6 +19,7 @@ plugins {
         plugin_data {
             database_type = "sqlite3"
             connection_string = "./.data/datastore.sqlite3"
```

Comment:
Should we be pedantic and rename this `rw_connection_string`? That would be noise in this commit, so I suggest doing it separately if we decide to do so.